delft3dworker/management/commands/containersync_sceneupdate.py

```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
     help = "sync containers with container and scene model"
 
     def handle(self, *args, **options):
```

Comment:
This method is a piece of art, truly.
- under 60 lines of code
- clearly readable
- solid reasoning